.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,9 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
This should be something like "To report a part of mocha not working as expected"